pkg/tracing/attributes.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file contains an implementation of an opentracing carrier built on top of the mixer's attribute bag. This allows
+// clients to inject their span metadata into the set of attributes its reporting to the mixer, and lets the mixer extract
+// span information propagated as attributes in a standard call to the mixer.
+
+package tracing
+
+import (
+	"context"
+	"strings"
+
+	"istio.io/mixer/pkg/attribute"
+)
+
+const (
+	prefix       = "istio-ot-"
```

Comment:
We'll need to elevate this to some global registry of known attribute names at some point...